modules/nf-core/modules/ratioconv/main.nf

```diff
@@ -0,0 +1,34 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process RATIOCONV {
+    //tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
+    container "ghcr.io/waseju/fiji:latest"
```

Comment:
Please migrate this and set up automated docker container builds and pushes to Dockerhub or ghcr at the `qbic-pipelines` account